rails/lib/tasks/db_feature_test_prepare.rake

```diff
@@ -17,12 +17,6 @@ namespace :db do
       ActiveRecord::Schema.verbose = false
       ActiveRecord::Tasks::DatabaseTasks.load_schema_for ActiveRecord::Base.configurations['feature_test'], :ruby, ENV['SCHEMA']
```

Comment:
@scytacki one of the deprecation warnings is around this line:

`This method was renamed to `#load_schema` and will be removed in the future. Use `#load_schema` instead.`  `load_schema` is available in 4.2 (https://apidock.com/rails/ActiveRecord/Tasks/DatabaseTasks/load_schema) - do you want to update that here in this PR so that it is in master?

Reply:
I tried, but they flipped the signatures from 4.2 to 5.0.  
In 4.2 `'load_schema` is `def load_schema(format = ActiveRecord::Base.schema_format, file = nil)`, but in 5.0 it is `def load_schema(configuration, format = ActiveRecord::Base.schema_format, file = nil, environment = env)`

so it doesn't look like we can switch to `load_schema` until we upgrade to 5.0